py-rse/bash-basics.Rmd

```diff
@@ -440,11 +440,14 @@ but we'll see some uses for it soon.
 > **Combining options**
 >
 > You'll occasionally need to use multiple options in the same command.
-> In these cases,
-> it's easier to combine them together behind a single dash:
+> In most command line tools,
+> multiple options can be combined with a single `-`
+> and no spaces between the options:
+>
 > ```shell
 > $ ls -Fa
 > ```
+>
```

Comment:
I would add a line saying that this isn't recommend practice because some commands take long-name options and it's very easy to mistake `-no` (meaning `-n -o`) with `--no` (meaning "answer 'no' to all questions").

Reply:
TIL...this.